src/CPUSolver.cpp

```diff
@@ -763,7 +725,7 @@ void CPUSolver::transferBoundaryFlux(int track_id,
     for (int p=0; p < _num_polar; p++) {
       track_out_flux(p,e) = track_flux(p,e) * bc;
       track_leakage(p,e) = track_flux(p,e) *
-                            _polar_weights(azim_index,p) * (!bc);
+                           _polar_weights(azim_index,p) * (!bc);
```

Comment:
Yes, multiplying by a boolean is fine in C/C++. Much simpler (and faster!) than using a conditional statement. Of course we could use ``1-bc`` instead. I'll change this here if both you and @samuelshaner think it necessary.